src/applications/common/schemaform/actions.js

```diff
@@ -82,7 +81,7 @@ export function setViewedPages(pageKeys) {
 function submitToUrl(body, submitUrl, trackingPrefix) {
   return new Promise((resolve, reject) => {
     const req = new XMLHttpRequest();
-    req.open('POST', `${environment.API_URL}${submitUrl}`);
+    req.open('POST', `${submitUrl}`);
```

Comment:
I think this can be `req.open('POST', submitURL);`

Reply:
ah yes, thanks